### PR TITLE
Fix documentation link

### PR DESCRIPTION
**Pull request by sambrightman**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -48,7 +48,7 @@ Links
     :target: https://readthedocs.org/projects/pydocstyle/?badge=latest
     :alt: Documentation Status
 
-* `Read the full documentation here <https://pydocstyle.org>`_.
+* `Read the full documentation here <http://pydocstyle.readthedocs.io>`_.
```
Comment (myint) [inline README.rst]: I think the `https` part can be left unchanged.

https://pydocstyle.readthedocs.io

Comment (Nurdok) [inline README.rst]: Actually, http://pydocstyle.org is the correct link.

**Comment by sambrightman** [comment]
Oh, sorry. Was unresponsive when I clicked it and the alternative came up first on Google.

**Comment by Nurdok** [comment]
If you correct the link to use `http` (instead of `https`) I'll merge this.

**Comment by sambrightman** [comment]
Done. Also suggested rtfd/readthedocs.org#2417.

**Comment by Nurdok** [comment]
Sorry if I was unclear, I meant you should change the link to http://pydocstyle.org/

**Comment by sambrightman** [comment]
Ah I see. HTTPS also does not work there... With `www` it serves HTTPS but with a bad certificate. I've re-pushed with HTTP and no `www` for now.

**Comment by Nurdok** [comment]
Merged. Thanks, @sambrightman!

**Pull request merged**